client/src/productOverview/image.jsx

```diff
@@ -118,7 +119,8 @@ const Image = (props) => {
             <img className="gallery-image" src={image.thumbnail_url}></img>
             {(imageURL === image.url) && <span className="blackLine"></span>}
           </div>)}
-          {(images.length > 7 && end < images.length) && <div className="arrow" onClick={() => handleScroll()}>﹀</div>}
+          {(images.length > 7 && start > 0) && <div className="arrow up" onClick={() => handleScroll('up')}>︿</div>}
```

Comment:
we have react-icons installed if you would prefer to use that instead: https://react-icons.github.io/react-icons/search?q=arrow